inspirehep/modules/migrator/ext.py

```diff
@@ -35,4 +36,5 @@ def __init__(self, app=None):
     def init_app(self, app):
         app.cli.add_command(migrate)
         app.cli.add_command(migrator)  # Deprecated
+        app.register_blueprint(blueprint)
```

Comment:
the problem registering the blueprint here is that it would be available in both ``app`` and ``api``, in our case we need the API only available in the ``api``. So the way to do this is remove the ``register_blueprint`` from here and add in the entrypoints:

```python
invenio_base.api_blueprints: [
 'inspire_migrator = inspirehep.modules.migrator.views:blueprint',
]

```